ci-operator/jobs/openshift/assisted-installer-agent/openshift-assisted-installer-agent-master-presubmits.yaml

```diff
@@ -6,17 +6,18 @@ presubmits:
     - ^master$
     - ^master-
     cluster: build02
-    context: ci/prow/e2e-metal-assisted
+    context: ci/prow/edge-e2e-metal-assisted
     decorate: true
     decoration_config:
       skip_cloning: true
     labels:
       ci-operator.openshift.io/cloud: packet-edge
       ci-operator.openshift.io/cloud-cluster-profile: packet-assisted
+      ci-operator.openshift.io/variant: edge
       ci.openshift.io/generator: prowgen
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
-    name: pull-ci-openshift-assisted-installer-agent-master-e2e-metal-assisted
-    rerun_command: /test e2e-metal-assisted
+    name: pull-ci-openshift-assisted-installer-agent-master-edge-e2e-metal-assisted
+    rerun_command: /test edge-e2e-metal-assisted
```

Comment:
Worth giving people heads-up that they need to do ``/test edge-<previous_name>`` when this PR gets in

Reply:
It was brought up in the arch meeting last week. I will make a team notification in Slack too.